graph-service/curator.py

```
@@ -0,0 +1,565 @@
+        
+    except Exception as e:
+        logger.error(f"Error analyzing graph: {e}", exc_info=True)
+        return None
+
+def consolidate_duplicate_relationships(driver, from_id: str, to_id: str, rel_type: str, rel_ids: List[str]):
+    """Consolidate duplicate relationships by keeping one and removing others."""
+    if len(rel_ids) <= 1:
+        return
+    
+    # Keep the first relationship, remove the rest
+    keep_id = rel_ids[0]
```

Ruff (0.14.7)

Local variable `keep_id` is assigned to but never used

Remove assignment to unused variable `keep_id`

(F841)